src/views/Rules/ListRules/index.jsx

```diff
@@ -249,9 +297,35 @@ function ListRules(props) {
       throw error;
     }
 
-    setRulesWithScheduledChanges(
-      rulesWithScheduledChanges.filter(i => i.rule_id !== dialogRule.rule_id)
-    );
+    if (Object.keys(dialogRule.requiredSignoffs).length > 0) {
+      // A change was scheduled, we need to update the card
+      // to reflect that.
+      try {
+        const sc = (await getScheduledChange(dialogRule.rule_id)).data
+          .scheduled_changes[0];
+
+        setRulesWithScheduledChanges(
+          rulesWithScheduledChanges.map(i => {
+            if (i.rule_id !== sc.rule_id) {
+              return i;
+            }
+
+            i.scheduledChange = sc;
```

Comment:
I think this will cause a `no-param-reassign` linting error. Alternatives:

1. Use `Object.assign`
2. Create a copy of the rule first then change it

```jsx
rulesWithScheduledChanges.map(r => {
    const rule = { ...r };
});
```

I think the second option is safer since we won't be modifying the original content of the array.

Reply:
:+1: It looks like `const rule = { ...r }` will create a shallow copy, but I think that will be OK?

Reply:
You are correct but yeah, I think that's okay for now.